vitest.config.ts

```diff
@@ -73,6 +73,35 @@ export default defineConfig({
           exclude: [...defaultExclude, '**/universal/**'],
           environment: 'nuxt',
           setupFiles: ['./test/setup-runtime.ts'],
+          environmentOptions: {
+            nuxt: {
+              overrides: {
+                routeRules: {
+                  '/specific-prerendered': { prerender: true },
+                  '/pre/test': { redirect: '/' },
+                  '/pre/**': { prerender: true },
+                },
+                experimental: {
+                  appManifest: process.env.TEST_MANIFEST !== 'manifest-off',
```

Comment:
feels like maybe we could move the unit tests outside of the test matrix entirely, in favour of just creating an additional project (you can leave this with me if you want)